<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rawImageViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>773</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="ImageWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Comport</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboComport"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDisconnect">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pixel Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboPixFormat">
         <item>
          <property name="text">
           <string>PIXFORMAT_RGB565</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PIXFORMAT_GRAYSCALE</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Framesize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboFramesize">
         <item>
          <property name="text">
           <string>FRAMESIZE_96X96</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editFilename"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSnapshot">
         <property name="text">
          <string>Snapshot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
